pyelftool_parser/src/analyzer.py

```diff
@@ -0,0 +1,203 @@
+import sys
```

Comment:
I think that you likely want the entire pyelftool to be in `tools/`

Reply:
I am not sure why I try to merge from my own repo. Then it keeps moving it to main folder. I think it is my command line problems. We could move it afterward after we decide how to integrate with system.